srv/components/provisioner/scripts/install.sh

```diff
@@ -436,12 +438,18 @@ install_cortx_pkgs()
     echo "Installed all Cortx packages successfully" | tee -a "${LOG_FILE}"
 }
 
-setup_bash_env()
+# setup_bash_env()
```

Comment:
Lets keep the function, nywaz we are not calling it

Reply:
done